Week3/homework/youAreAmazing.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+{
+    function giveCompliment(){
+        let myName = "Yalcin";
+        let myArr = ["cool", "awesome", "legend", "great", "pretty", "smart", "hardworking", "patient", "respected", "handsome"];
```

Comment:
`const` instead of `let` here please. Also, would suggest to pick a better name for variable `myArr`. It doesn't really describe what information it is pointing to. 🙂